scripts/install.ps1

```diff
@@ -46,4 +47,5 @@ Remove-Item -Force -Recurse -Path "$extractDir"
 Write-Output "Tilt installed!"
 Write-Output "Run '$dest up' to start."
 Write-Output "Or add $binDir to your PATH"
+tilt verify-install
```

Comment:
this probably needs to be $dest, since, as the comment above notes, this assumes tilt is on your path

(though tbh it might be easier for me to just make this change myself so i can test it on windows if you want to back this out)

Reply:
Ohhh right...
Yeah, that would actually be great if you could change it on your end and test it (since i don't have access to a Windows machine 😅)